rust/Demo/led/src/main.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#![no_main]
+#![no_std]
+
+extern crate panic_halt as _;
+
+use core::fmt::{self, Write};
+use embedded_hal::blocking::delay::DelayMs;
+use riscv::delay::McycleDelay;
+use riscv_rt::entry;
+use riscv_semihosting::hio;
+use IBEX::Peripherals;
+
+const CPU_CLOCK_HZ: u32 = 50_000_000;
+
+#[entry]
+fn main() -> ! {
+    let mut delay = McycleDelay::new(CPU_CLOCK_HZ);
+    let p = Peripherals::take().unwrap();
+
+    // We can either log via uart, or JTAG/openocd.
+    let mut uart = Uart::new(p.UART0);
+    // let mut _jtag_log = hio::hstdout().unwrap();
```

Comment:
Does this commented out line need to be here? If removing, remember to remove the `hio` import.